### PR TITLE
[ES] Update `Client/Beatmap editor/Compose`

### PR DESCRIPTION
**Pull request by UnknownOsu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 1]
```diff
@@ -1,276 +1,203 @@
...
+*Para una visión general del proceso de mapping, véase: [Beatmapping](/wiki/Beatmapping)*
 
-Para una guía de novatos: [Beatmapping](/wiki/Beatmapping).
+### Línea de tiempo de los objetos
 
-(Comenzando desde izquierda a derecha y de arriba hacia abajo)
+![](img/objects-timeline.jpg "Línea de tiempo de los objetos con respecto al divisor de ritmo y la marca de tiempo")
 
-### Esquina superior izquierda (Linea de Tiempo de los Objetos golpeables)
+La línea de tiempo se puede acercar y alejar con los botones `+`/`-` a la izquierda o, alternativamente, desplazándose con la tecla `Alt` mantenida. Las dos líneas blancas en el medio indican la marca de tiempo actual. Además, los objetos en la línea de tiempo se pueden seleccionar y mover con el botón izquierdo del ratón, o eliminarlos haciendo clic con el botón derecho.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:23]:
```suggestion
La línea de tiempo se puede acercar y alejar con los botones `+`/`-` de la izquierda o, alternativamente, desplazándose con la tecla `Alt` mantenida. Las dos líneas blancas en el medio indican la marca de tiempo actual. Además, los objetos en la línea de tiempo se pueden seleccionar y mover con el botón izquierdo del ratón, o eliminarlos haciendo clic con el botón derecho.
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 2]
```diff
@@ -1,276 +1,203 @@
...
+La **pestaña compose** del [editor de beatmaps](/wiki/Client/Beatmap_editor) es el lugar donde los [mappers](/wiki/Beatmapping) pasan la mayor parte de su tiempo trabajando en beatmaps después de ajustar la [temporización](/wiki/Beatmapping/Timing). Los [patrones de los objetos](/wiki/Beatmap/Pattern), [hitsounds](/wiki/Beatmapping/Hitsound) y otros aspectos de diseño del beatmap se pueden inspeccionar en esta pestaña.
 
-**Componer** es el lugar donde un [mapper](/wiki/Beatmapping) pasa la mayor parte del tiempo mapeando su diseño con respecto a la línea de tiempo después de configurar las Secciones de BPM y Sincronización. Este es también el lugar donde los jugadores pueden practicar una dificultad clasificada e inspeccionar su diseño similar al modo de entrenamiento de DDR.
+El editor comparte las mismas herramientas entre osu!, osu!taiko y osu!catch, mientras que osu!mania tiene un diseño de la pestaña compose específico más adecuado para el modo de juego. Se puede acceder cambiando el [allowed mode](/wiki/Client/Beatmap_editor/Song_setup#advanced) de la dificultad a `osu!mania`.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:13]:
```suggestion
El editor comparte las mismas herramientas entre osu!, osu!taiko y osu!catch, mientras que osu!mania tiene un diseño de la pestaña compose específico más adecuado para ese modo de juego. Se puede acceder cambiando el [allowed mode](/wiki/Client/Beatmap_editor/Song_setup#advanced) de la dificultad a `osu!mania`.
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 3]
```diff
@@ -1,276 +1,203 @@
...
-### Esquina superior derecha (Divisor de ritmo)
+| [Divisor de ritmo](/wiki/Client/Beatmap_editor/Beat_snap_divisor) | Limita la cantidad de marcas de la línea de tiempo que determinan en qué subdivisiones de un pulso se pueden colocar los objetos. Si se mueve el control deslizante hacia la derecha, aumenta la granularidad de las marcas de la línea de tiempo. |
+| `Insertar descanso` | Agrega un [descanso](/wiki/Beatmap/Break) en la marca de tiempo actual. |
+| x/y | Muestra la posición de un objeto seleccionado en el campo de juego o, si no hay nada seleccionado, la posición del cursor. |
 
-![Divisor de ritmo](img/beat-snap-divisor.jpg "Divisor de ritmo")
+Mantén presionado `Alt` para cambiar el control deslizante al modo de [ajuste de distancia](/wiki/Client/Beatmap_editor/Distance_snap) de las notas. El multiplicador de distancia puede variar entre 0,1x y 6,0x.
 
-**En términos sencillos, ajusta la longitud dada de un hilo de coser de color (sección de temporización) para un espacio de diseño de bordado (medidas, divididas por el divisor para llamarlas beats) en la tela (canción).**
+### Barra de herramientas izquierda
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:37]:
```suggestion
### Barra de herramientas de la izquierda
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 4]
```diff
@@ -1,276 +1,203 @@
...
 
-Mant `Alt` para activar el Divisor de ritmo. El valor puede variar de x0.1 a x2.0.
+![](img/tools.jpg "Barra de herramientas izquierda")
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:39]:
```suggestion
![](img/tools.jpg "Barra de herramientas de la izquierda")
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 5]
```diff
@@ -1,276 +1,203 @@
...
-| Adiciones | Auto, Normal, Soft, Drum | Básicamente lo mismo de arriba, PERO la configuración de Adiciones sobrescribe la configuración del conjunto de muestras. Ej: Temporización **Drum 55%**, conjunto de muestras **Soft**, mientras que estas Adiciones es **Normal**, reproducirá todo el sonido **Normal** relacionado en las notas como clap, normal, finish, whistle con 55% de volumen. |
-| Seleccionar (`1`) | **Clic izquierdo/ arrastrar:** Mueve la marda de tiempo y ubicación de una nota. **Clic derecho:** Elimina la nota. **`Ctrl` + Clic:** Selección múltiple. | Selecciona y modifica las notas existentes. |
-| Circulo (`2`) | **Clic Izquierdo/Derecho** Coloca/Elimina la nota. | Agrega una nota en la columna actual en función de la posición del mouse. |
-| Mantener (`3`) | **Clic izquierdo (Mantener):** Coloca una nota sostenida y ajusta su duración. **Sotarla:** Establece longitud de retención | Agrega un nota de sostenida en la columna actual según la posición del mouse. |
+### Barra de herramientas izquierda
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:128]:
```suggestion
### Barra de herramientas de la izquierda
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 6]
```diff
@@ -1,276 +1,203 @@
...
+La **pestaña compose** del [editor de beatmaps](/wiki/Client/Beatmap_editor) es el lugar donde los [mappers](/wiki/Beatmapping) pasan la mayor parte de su tiempo trabajando en beatmaps después de ajustar la [temporización](/wiki/Beatmapping/Timing). Los [patrones de los objetos](/wiki/Beatmap/Pattern), [hitsounds](/wiki/Beatmapping/Hitsound) y otros aspectos de diseño del beatmap se pueden inspeccionar en esta pestaña.
 
-**Componer** es el lugar donde un [mapper](/wiki/Beatmapping) pasa la mayor parte del tiempo mapeando su diseño con respecto a la línea de tiempo después de configurar las Secciones de BPM y Sincronización. Este es también el lugar donde los jugadores pueden practicar una dificultad clasificada e inspeccionar su diseño similar al modo de entrenamiento de DDR.
+El editor comparte las mismas herramientas entre osu!, osu!taiko y osu!catch, mientras que osu!mania tiene un diseño de la pestaña compose específico más adecuado para ese modo de juego. Se puede acceder cambiando el [allowed mode](/wiki/Client/Beatmap_editor/Song_setup#advanced) de la dificultad a `osu!mania`.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:13]:
```suggestion
El editor comparte las mismas herramientas entre osu!, osu!taiko y osu!catch, mientras que osu!mania tiene un diseño de la pestaña compose específico más adecuado para ese modo de juego. Se puede acceder cambiando el [modo permitido](/wiki/Client/Beatmap_editor/Song_setup#advanced) de la dificultad a `osu!mania`.
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 7]
```diff
@@ -1,276 +1,203 @@
...
+*Para una visión general del proceso de mapping, véase: [Beatmapping](/wiki/Beatmapping)*
 
-Para una guía de novatos: [Beatmapping](/wiki/Beatmapping).
+### Línea de tiempo de los objetos
 
-(Comenzando desde izquierda a derecha y de arriba hacia abajo)
+![](img/objects-timeline.jpg "Línea de tiempo de los objetos con respecto al divisor de ritmo y la marca de tiempo")
 
-### Esquina superior izquierda (Linea de Tiempo de los Objetos golpeables)
+La línea de tiempo se puede acercar y alejar con los botones `+`/`-` de la izquierda o, alternativamente, desplazándose con la tecla `Alt` mantenida. Las dos líneas blancas en el medio indican la marca de tiempo actual. Además, los objetos en la línea de tiempo se pueden seleccionar y mover con el botón izquierdo del ratón, o eliminarlos haciendo clic con el botón derecho.
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:23]:
```suggestion
La línea de tiempo se puede acercar y alejar con los botones `+`/`-` de la izquierda o, alternativamente, desplazándose con la tecla `Alt` mantenida. Las dos líneas blancas en el centro indican la marca de tiempo actual. Además, los objetos en la línea de tiempo se pueden seleccionar y mover con el botón izquierdo del ratón, o eliminarlos haciendo clic con el botón derecho.
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 8]
```diff
@@ -1,276 +1,203 @@
...
 | :-- | :-- | :-- |
-| Whistle (`W`) | ("Seleccionar" en una nota de golpeo) **Clic izquierdo:** Usar sonido de silbido en esta nota | Agrega una adición de sonido de silbido a la selección. |
-| Finish (`E`) | ("Seleccionar" en una nota de golpeo) **Clic izquierdo:** Usar sonido de finalización en esta nota | Agrega un sonido final (platillos) a la selección. |
-| Clap (`R`) | ("Seleccionar" en una nota de golpeo) **Clic izquierdo:** Usa un sonido de aplauso en esta nota | Agrega una adición de sonido de aplauso a la selección. |
-
-#### Herramientas de ayuda
+| `Grid Snap` (`T`) | Al mantener pulsado `Mayús`: Alterna temporalmente. | Ajusta objetos [a la cuadrícula](/wiki/Beatmapping/Grid_snapping) mientras se mueven. |
+| `Distance Snap` (`Y`) | Al mantener pulsado `Alt`: Alterna temporalmente, cambiando el divisor de ritmo a ajuste de distancia de las notas. `Alt` + `Rueda del ratón`: Ajusta el multiplicador del ajuste de distancia de las notas. | Calcula la [distancia](/wiki/Client/Beatmap_editor/Distance_snap) entre objetos consecutivos en función de su distancia temporal. Es mejor usarlo mientras la línea de tiempo está en pausa. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:75]:
```suggestion
| `Grid Snap` (`T`) | Mantener pulsado `Mayús`: Alterna temporalmente. | Ajusta objetos [a la cuadrícula](/wiki/Beatmapping/Grid_snapping) mientras se mueven. |
| `Distance Snap` (`Y`) | Mantener pulsado `Alt`: Alterna temporalmente, cambiando el divisor de ritmo a ajuste de distancia de las notas. `Alt` + `Rueda del ratón`: Ajusta el multiplicador del ajuste de distancia de las notas. | Calcula la [distancia](/wiki/Client/Beatmap_editor/Distance_snap) entre objetos consecutivos en función de su distancia temporal. Es mejor usarlo mientras la línea de tiempo está en pausa. |
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 9]
```diff
@@ -1,276 +1,203 @@
...
-![Area de juego de osu!mania](img/playfield-mania.jpg "Area de juego osu!mania")
-
-**Representación visual del mappeo basada en la marca de tiempo actual.**
+| Botón (atajo de teclado) | Uso |
+| :-- | :-- |
+| `Sampleset` | Reemplaza el [conjunto de muestras](/wiki/Beatmapping/Sampleset) de los objetos seleccionados (incluidos sus hitnormals). Al elegir `Auto`, se restablece el conjunto de muestras al del [punto de tiempo](/wiki/Client/Beatmap_editor/Timing#puntos-de-tiempo) activo. |
+| `Additions` | Reemplaza el conjunto de muestra de los objetos seleccionados, afectando solo sus hitsounds adicionales (whistle, finish y clap). Al elegir `Auto`, se restablece el conjunto de muestras al del punto de tiempo activo. |
+| `Select` (`1`) | `Clic izquierdo` o `Arrastrar hacia la izquierda`: Mueve la marca de tiempo y ubicación de la nota. `Clic derecho`: Elimina la nota. `Ctrl` + `Clic izquierdo`: Selecciona varios objetos. |
+| `Circle` (`2`) | `Clic izquierdo`: Coloca una nota. |
+| `Hold` (`3`) | Al mantener pulsado `Clic izquierdo`: Coloca una nota larga y ajusta su longitud. Suelta el clic para finalizar la nota. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:138]:
```suggestion
| `Hold` (`3`) | Mantener pulsado `Clic izquierdo`: Coloca una nota larga y ajusta su longitud. Suelta el clic para finalizar la nota. |
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 10]
```diff
@@ -1,276 +1,203 @@
...
 
-El botón de **Insertar descanso** iniciará / detendrá el "descanso" en la marca de tiempo actual y **x / y** es la coordenada actual del mouse.
-
-### Centro izquierda (Selector)
-
-![Hit Selector de notas](img/tools.jpg "Hit Selector de notas")
+| Botón (atajo de teclado) | Descripción |
+| :-- | :-- |
+| `Sampleset` | Reemplaza el [conjunto de muestras](/wiki/Beatmapping/Sampleset) de los objetos seleccionados (incluidos sus hitnormals). Al elegir `Auto`, se restablece el conjunto de muestras al del [punto de tiempo](/wiki/Client/Beatmap_editor/Timing#puntos-de-tiempo) activo. |
+| `Additions` | Reemplaza el conjunto de muestra de los objetos seleccionados, afectando solo sus hitsounds adicionales (whistle, finish y clap). Al elegir `Auto`, se restablece el conjunto de muestras al del punto de tiempo activo. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:44]:
```suggestion
| `Sampleset` | Reemplaza el [conjunto de muestras](/wiki/Beatmapping/Sampleset) de los objetos seleccionados (incluidos sus hitnormals). Elegir `Auto` restablecerá el conjunto de muestras al del [punto de tiempo](/wiki/Client/Beatmap_editor/Timing#puntos-de-tiempo) activo. |
| `Additions` | Reemplaza el conjunto de muestra de los objetos seleccionados, afectando solo sus hitsounds adicionales (whistle, finish y clap). Elegir `Auto` restablecerá el conjunto de muestras al del punto de tiempo activo. |
```

File: wiki/Client/Beatmap_editor/Compose/es.md [hunk 11]
```diff
@@ -1,276 +1,203 @@
...
-### Centro (Area de juego)
+![](img/tools-mania.jpg "Panel de objetos para osu!mania")
 
-![Area de juego de osu!mania](img/playfield-mania.jpg "Area de juego osu!mania")
-
-**Representación visual del mappeo basada en la marca de tiempo actual.**
+| Botón (atajo de teclado) | Uso |
+| :-- | :-- |
+| `Sampleset` | Reemplaza el [conjunto de muestras](/wiki/Beatmapping/Sampleset) de los objetos seleccionados (incluidos sus hitnormals). Al elegir `Auto`, se restablece el conjunto de muestras al del [punto de tiempo](/wiki/Client/Beatmap_editor/Timing#puntos-de-tiempo) activo. |
+| `Additions` | Reemplaza el conjunto de muestra de los objetos seleccionados, afectando solo sus hitsounds adicionales (whistle, finish y clap). Al elegir `Auto`, se restablece el conjunto de muestras al del punto de tiempo activo. |
```
Comment (agatemosu) [inline wiki/Client/Beatmap_editor/Compose/es.md:135]:
```suggestion
| `Sampleset` | Reemplaza el [conjunto de muestras](/wiki/Beatmapping/Sampleset) de los objetos seleccionados (incluidos sus hitnormals). Elegir `Auto` restablecerá el conjunto de muestras al del [punto de tiempo](/wiki/Client/Beatmap_editor/Timing#puntos-de-tiempo) activo. |
| `Additions` | Reemplaza el conjunto de muestra de los objetos seleccionados, afectando solo sus hitsounds adicionales (whistle, finish y clap). Elegir `Auto` restablecerá el conjunto de muestras al del punto de tiempo activo. |
```

**Review by agatemosu — APPROVED** [review]

**Review by agatemosu — APPROVED** [review]

**Comment by agatemosu** [comment]
no me di cuenta (mucho rojo verde azul gris)

**Review by TicClick — APPROVED** [review]

**Pull request merged**